### DISCUSSION TITLE
Что случилось с vk sans?

### DISCUSSION THREAD

**Discussion by ivanmem** [VKCOM/VKUI#4201] [Q&A] [answered]
Этого шрифта больше нет в проекте.  Почему?
Если его удалили из-за лицензии - о какой лицензии идёт речь?
Как быть со старыми проектами где он используется? 
Можно ли использовать этот шрифт в своих проектах просто скопировав его с v4?

**Comment by inomdzhon** [comment, marked as answer]
@xeleoss ещё раз привет!

Шрифт в **VKUI v5** больше недоступен, но ты можешь использовать шрифты `VK_Sans_Display_Medium.woff` и `VK_Sans_Display_Medium.woff2`, взяв их из **VKUI v4** (см. https://github.com/VKCOM/VKUI/tree/v4/src/fonts).

> Как быть со старыми проектами где он используется?

Если мы говорим о `VK_Sans_Display_Medium.woff` и `VK_Sans_Display_Medium.woff2`, то никаких действий не требуется.

**Reply by dimstr** [reply]
> @xeleoss ещё раз привет!
> 
> Шрифт в **VKUI v5** больше недоступен, но ты можешь использовать шрифты `VK_Sans_Display_Medium.woff` и `VK_Sans_Display_Medium.woff2`, взяв их из **VKUI v4** (см. https://github.com/VKCOM/VKUI/tree/v4/src/fonts).
> 
> > Как быть со старыми проектами где он используется?
> 
> Если мы говорим о `VK_Sans_Display_Medium.woff` и `VK_Sans_Display_Medium.woff2`, то никаких действий не требуется.

Здравствуйте, подскажите а из-за чего данный шрифт был удалён из VKUI v5?

**Reply by inomdzhon** [reply]
Приветствую!

В будущем шрифт будет использоваться для наших внутренних целей.
